src/Tribe/Templates.php

```diff
@@ -60,4 +60,19 @@ public static function locate_stylesheet( $stylesheets, $fallback = false ) {
 		return $located;
 	}
 
+	/**
+	 * Add our own method is_embed to check by WordPress Version and function is_embed
+	 * to prevent fatal errors in WordPress 4.3 and earlier
+	 *
+	 * @version 4.2.1
+	 */
+	public static function is_embed() {
+		global $wp_version;
+		if ( version_compare( $wp_version, '4.4', '<' ) || ! function_exists( 'is_embed' ) ) {
+			return false;
+		} else {
```

Comment:
The `else` is superfluous. You can just `return is_embed();` outside of the control structure.